exporter/importables.go

```diff
@@ -1202,6 +1202,7 @@ var resourcesMap map[string]importable = map[string]importable{
 				})
 			}
 
+			// TODO: it's not completely correct condition - we need to make emit smarter - emit only if permissions are different...
```

Comment:
Is this comment meant for the block before this?
What do you mean by "permissions are different"? Different from what exactly?

Reply:
Ah I understand. Could you just change your comment to this answer? It is much clearer :)

Reply:
done